.scripts/package.sh

```diff
@@ -260,13 +260,25 @@ generate_swift_package () {
     template_replace $package "// GENERATE BINARIES" $binaries; rm -f $binaries
 }
 
+login_default() {
+    echo "$GITHUB_TOKEN" | gh auth login --with-token
```

Comment:
Are these printing the tokens? Should those be obscured?

Reply:
It shouldn't. We're piping echo to `gh`, so won't be logged to `stdout`